### ISSUE TITLE
accessibility* props are not available for backdrop view

### ISSUE THREAD

**Issue by Aure77** [react-native-modal/react-native-modal#699]
`ModalProps` extends `ViewProps` that should contains all accessibility props (https://reactnative.dev/docs/accessibility), but they are not used/applied to any view. 
As backdrop is pressable, we should have ability to set accessibility attributes on this view.
For example:
```javascript
    <Modal
      isVisible={visible}
      hasBackdrop={true}
      onBackdropPress={onBackButtonPress}
      accessible
      accessibilityRole="button"
      accessibilityLabel="Close modal"
      ...
```

But they are not taken into account because only `onPress` is passed here:
https://github.com/react-native-modal/react-native-modal/blob/b580105f4b2b4e1163bed1228f241b16ea17dacb/src/modal.tsx#L758

So I might be useful to add this accessibility attributes to backdrop view to help disabled people.
(maybe prefix _accessib*_ props with _backdropAccessib*_ to avoid concern confusion)

PS: I know there is _customBackdrop_, but I will be helpful to be handled out of the box by default backdrop

**Comment by Aure77** [comment]
any updates about accessibility ?

Here is my workaround:

```diff
diff --git a/node_modules/react-native-modal/dist/modal.d.ts b/node_modules/react-native-modal/dist/modal.d.ts
index b63bcfc..6029d06 100644
--- a/node_modules/react-native-modal/dist/modal.d.ts
+++ b/node_modules/react-native-modal/dist/modal.d.ts
@@ -26,6 +26,7 @@ declare const defaultProps: {
     backdropOpacity: number;
     backdropTransitionInTiming: number;
     backdropTransitionOutTiming: number;
+    backdropAccessibilityLabel: string | undefined;
     customBackdrop: React.ReactNode;
     useNativeDriver: boolean;
     deviceHeight: number | null;
@@ -62,7 +63,7 @@ export declare type ModalProps = ViewProps & {
     onOrientationChange?: OnOrientationChange;
     presentationStyle?: PresentationStyle;
     useNativeDriverForBackdrop?: boolean;
-} & typeof defaultProps;
+} & Partial<typeof defaultProps>;
 export declare class ReactNativeModal extends React.Component<ModalProps, State> {
     static propTypes: {
         animationIn: PropTypes.Requireable<string | object>;
@@ -74,6 +75,7 @@ export declare class ReactNativeModal extends React.Component<ModalProps, State>
         hasBackdrop: PropTypes.Requireable<boolean>;
         backdropColor: PropTypes.Requireable<string>;
         backdropOpacity: PropTypes.Requireable<number>;
+        backdropAccessibilityLabel: PropTypes.Requireable<string>;
         backdropTransitionInTiming: PropTypes.Requireable<number>;
         backdropTransitionOutTiming: PropTypes.Requireable<number>;
         customBackdrop: PropTypes.Requireable<PropTypes.ReactNodeLike>;
diff --git a/node_modules/react-native-modal/dist/modal.js b/node_modules/react-native-modal/dist/modal.js
index 80f4e75..3b72366 100644
--- a/node_modules/react-native-modal/dist/modal.js
+++ b/node_modules/react-native-modal/dist/modal.js
@@ -398,7 +398,7 @@ export class ReactNativeModal extends React.Component {
                 !React.isValidElement(this.props.customBackdrop)) {
                 console.warn('Invalid customBackdrop element passed to Modal. You must provide a valid React element.');
             }
-            const { customBackdrop, backdropColor, useNativeDriver, useNativeDriverForBackdrop, onBackdropPress, } = this.props;
+            const { customBackdrop, backdropColor, useNativeDriver, useNativeDriverForBackdrop, onBackdropPress, backdropAccessibilityLabel, } = this.props;
             const hasCustomBackdrop = !!this.props.customBackdrop;
             const backdropComputedStyle = [
                 {
@@ -418,7 +418,7 @@ export class ReactNativeModal extends React.Component {
             }
             // If there's no custom backdrop, handle presses with
             // TouchableWithoutFeedback
-            return (React.createElement(TouchableWithoutFeedback, { onPress: onBackdropPress }, backdropWrapper));
+            return (React.createElement(TouchableWithoutFeedback, { onPress: onBackdropPress, accessibilityLabel: backdropAccessibilityLabel, accessibilityRole: "button", accessible: true }, backdropWrapper));
         };
         const { animationIn, animationOut } = buildAnimations(extractAnimationFromProps(props));
         this.animationIn = animationIn;
```

**Comment by virginiab10** [comment]
+1

**Comment by kannan-rjp** [comment]
@Aure77 Is this ok with iOS?
